test/unit/ui/app/actions.spec.js

```diff
@@ -75,7 +75,7 @@ describe('Actions', function () {
 
     it('calls submitPassword and verifySeedPhrase', async function () {
 
-      const store = mockStore({})
+      const store = mockStore({ metamask: {} })
```

Comment:
Is it time for a default?

Reply:
Yes, I think so. I've created #8437 for this.

Reply:
#8437 has been merged, and this has been rebased onto `develop`, so now these changes aren't required anymore.